utf8_decode/src/lib.rs

```diff
@@ -1,76 +1,126 @@
-#![cfg_attr(not(feature = "metadata"), no_std)]
+use crate::proc_block_v1::*;
+use hotg_rune_proc_blocks::{
+    ndarray, runtime_v1::*, string_tensor_from_ndarray, BufferExt,
+};
+
+wit_bindgen_rust::export!("../wit-files/rune/proc-block-v1.wit");
+
 #[macro_use]
 extern crate alloc;
-
-use alloc::{borrow::Cow, string::ToString};
-use hotg_rune_proc_blocks::{ProcBlock, Tensor, Transform};
+use alloc::string::ToString;
```

Comment:
We're pulling in the standard library so we don't need to explicitly link with `alloc` or import `ToString` any more.